src/wallet/bdb.cpp

```diff
@@ -716,6 +728,13 @@ std::unique_ptr<DatabaseCursor> BerkeleyBatch::GetNewCursor()
     return std::make_unique<BerkeleyCursor>(m_database);
 }
 
+std::unique_ptr<DatabaseCursor> BerkeleyBatch::GetNewPrefixCursor(CDataStream& prefix)
+{
+    if (!pdb) return nullptr;
+    Assume(!prefix.empty());
+    return std::make_unique<BerkeleyCursor>(m_database, /*batch=*/nullptr, prefix);
```

Comment:
Note:
If this is used in a read-write process, I'm quite sure that it will suffer from the same problematic tackled in #27556.

Reply:
Done